src/Utilities/Blas.cpp

```diff
@@ -0,0 +1,18 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Utilities/Blas.hpp"
+
+namespace {
+extern "C" {
+#ifdef DISABLE_OPENBLAS_MULTITHREADING
+void openblas_set_num_threads(int num_threads);
```

Comment:
Is there a reason to declare this ourselves instead of including cblas.h?

Reply:
I added a comment to the code. Our `Blas` library doesn't provide include directories, so `cblas.h` might not be available, or it might be inconsistent with the BLAS library that we're linking in. The reason we don't have the include directories is that CMake's FindBLAS (https://cmake.org/cmake/help/latest/module/FindBLAS.html) doesn't provide it. From what I found they don't bother providing include directories because everyone declares the BLAS functions anyway.